table/block_based/block_based_table_reader.cc

```diff
@@ -594,13 +594,12 @@ Status BlockBasedTable::Open(
 
   // From read_options, retain deadline, io_timeout, and rate_limiter_priority.
   // In future, we may retain more
-  // options. Specifically, w ignore verify_checksums and default to
-  // checksum verification anyway when creating the index and filter
-  // readers.
+  // options.
```

Comment:
Since no reason was provided in the comment for dropping any of the options (besides the `verify_checksums`, which you just added back), can we try using `read_options` instead of `ro`?